src/pybind/mgr/dashboard/ci/cephadm/bootstrap-cluster.sh

```diff
@@ -25,7 +25,7 @@ fsid=$(cat /etc/ceph/ceph.conf | grep fsid | awk '{ print $3}')
 cephadm_shell="cephadm shell --fsid ${fsid} -c /etc/ceph/ceph.conf -k /etc/ceph/ceph.client.admin.keyring"
 
 {% for number in range(1, nodes) %}
-  ssh-copy-id -f -i /etc/ceph/ceph.pub  -o StrictHostKeyChecking=no root@{{ prefix }}-node-0{{ number }}
+  ssh-copy-id -f -i /etc/ceph/ceph.pub  -o StrictHostKeyChecking=no root@192.168.100.10{{ number }}
```

Comment:
Why not using hostnames instead of IP addresses?

Reply:
hostnames are not getting recognised with fedora 36. I haven't debugged why is it so.

Reply:
i was testing out with ip address in the jenkins with this PR and it seems to work